src/promptflow/promptflow/_trace/_start_trace.py

```diff
@@ -189,3 +191,32 @@ def _determine_trace_url(
     elif session_configured and session_id is not None:
         ui_url += f"?session={session_id}"
     return ui_url
+
+
+def _get_workspace_triad() -> typing.Optional[AzureMLWorkspaceTriad]:
+    # TODO: we have no conclusion on the local to cloud experience yet
+    #       one might be experience is `pf config xxx`, and we should get the triad via config;
+    #       before that, use below environ name temporarily
+    subscription_id = os.getenv("pf_test_subscription_id", None)
```

Comment:
Let's only merge this PR when we have the pf config ready?

Reply:
`pf config set trace.provider=azureml:/subscriptions/<subscription_id>/resourceGroups/<resource_group>/providers/Microsoft.MachineLearningServices/workspaces/<workspace_name>`